aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -299,83 +299,6 @@ static inline void checkFloatingOrComplex(const Tensor& t, const char* const f_n
               f_name, ": Expected a floating point or complex tensor as input. Got ", toString(t.scalar_type()));
 }
 
-/*
```

Comment:
What's the benefit of moving the code from here to `aten/src/ATen/native/BatchLinearAlgebra.cpp` instead of introducing the changes in this file?

Reply:
Now there's just one entry point to this function (through `_linalg_check_errors`) so that way it's impossible to use wrong and we always get something that's composite compliant and all that.